controllers/article-controllers.js

```diff
@@ -29,3 +35,13 @@ exports.patchArticle = (req, res, next) => {
     })
     .catch(next);
 };
+
+exports.getComments = (req, res, next) => {
+  const { article_id } = req.params;
+
+  Promise.all([checkArticleExists(article_id), checkCommentsExist(article_id), fetchComments(article_id)])
```

Comment:
You are already checking that the article exists 👍 Please get a test in to assert the behaviour this exhibits!
Also, there isn't really a need to check if the comments exist - if you get an empty array back, that's when we'll want to check the existence of the article

Reply:
I have tests that check when an article_id doesn't exist, a relevant error is provided. This shows that the checkArticleExists function works, and rejects a promise when it receives a response from the db.query which has an empty array on the .rows property.